### PR TITLE
Enumerable

### PR DESCRIPTION
**Pull request by brytebee**
## The Enums
In this PR, we have accomplished the following:
- setup the project 
- install rubocop linter check file
- created Ruby class
- created Ruby module
- simulated the enumerable methods of Ruby

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-In this project, we have created a simple app to decode morse code. We engineered the app using Ruby.
+In this project, we have created a class my_list and a module enumerable. The module implements a subset of the functionality of Enumerable.
 
 ## Screenshot
```
Comment (t-yanick) [inline README.md]: - [OPTIONAL] Please the link to your screenshot is broken, kindly fix that as it will make your readme file look more professional.

**Review by t-yanick — APPROVED** [review]
Hi @, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (brytebee) [reply README.md]: Done.

**Comment by brytebee** [comment]
Hello, @t-yanick 👋👋,

Thanks for the warm-hearted review 👀👀.



I'll go ahead to merge and move on to the next milestone 🏁🏁.

**_Happy Hacking 😄😄_**

**Pull request merged**